packages/three-vrm/src/vrm/humanoid/VRMHumanoidImporter.ts

```diff
@@ -30,7 +30,7 @@ export class VRMHumanoidImporter {
     if (schemaHumanoid.humanBones) {
       await Promise.all(
         schemaHumanoid.humanBones.map(async (bone) => {
-          if (!bone.bone || !bone.node) {
+          if (!bone.bone || typeof bone.node !== 'number') {
```

Comment:
Fix unexpected return when `bone.node` is `0`.

Reply:
ohhhhhh

personally prefer this:

```suggestion
          if (!bone.bone || bone.node == null) {
```